src/Account.ts

```diff
@@ -12,6 +17,40 @@ export class Account {
     readonly encryptedKeyStore: EncryptedKeystoreV3Json,
   ) { }
 
+  /**
+   * @param {Web3} web3 The web3 instance.
+   * @param password The password required to unlock the account.
+   * @returns {{account: Account; encryptedAccount: PrivateKey}}
+   */
+  public static create(web3: Web3, password: string): any {
+    const account = Account.newWeb3Account(web3);
+    const encryptedAccount = Account.encrypt(account, web3, password);
```

Comment:
Do we need `newWeb3Account` and `encrypt` function?
It is not been called from anywhere other than `create` function.
We can remove these two private functions

Reply:
Removed `newWeb3Account` and `encrypt` private functions.